### PR TITLE
More API v2

### PR DESCRIPTION
**Pull request by rowanseymour**
- get_campaigns
- get_campaign_events
- get_channels
- get_channel_events

### CODE REVIEW

**Comment by rowanseymour** [comment]
@norkans7

File: temba_client/v2/__init__.py [hunk 1]
```diff
@@ -5,7 +5,9 @@
 that users continue using the existing API v1.
 """
 
-from .types import Broadcast, Contact, Field, Group, Label, Message, Org, Run
+from .types import (
+    Broadcast, Campaign, CampaignEvent, Channel, ChannelEvent, Contact, Field, Group, Label, Message, Org, Run
```
Comment (norkans7) [inline temba_client/v2/__init__.py:9]: I guess we prefered multiline imports

File: test_files/v2/channels.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            "country": "RW",
+            "device": {
+                "name": "Nexus 5X",
+                "power_level": 99,
+                "power_status": "STATUS_DISCHARGING",
+                "power_source": "BATTERY",
+                "network_type": "WIFI"
+            },
+            "last_seen": "2016-03-01T05:31:27.456000Z",
+            "created_on": "2014-06-23T09:34:12.866000Z"
```
Comment (norkans7) [inline test_files/v2/channels.json:18]: On RapidPro I think we need to expose the channel type

Comment (norkans7) [inline test_files/v2/channels.json:18]: probably the role as well

Comment (rowanseymour) [inline test_files/v2/channels.json:18]: Yeah for completeness that would be good (tho you do wonder if anyone has ever used this endpoint...)

Comment (norkans7) [inline test_files/v2/channels.json:18]: probably no one uses it, after all those who use API always like all details, And yes you can remove it until someone request it

**Pull request merged**